cortx-motr.spec.in

```diff
@@ -1,3 +1,4 @@
+%undefine _missing_build_ids_terminate_build
```

Comment:
Why?

Reply:
This to  avoid  some  rpm compilation error

Reply:
What error?

Reply:
missing build id

Reply:
If the build id is missing - it should complain with an error, shouldn't it?

Reply:
yea, I do not know how to resolve, the only way i come to know is by putting the above statements

Reply:
Wrap it as a platform-specific thing, then.